src/Microsoft.AspNetCore.Blazor.Razor.Extensions/BlazorDesignTimeNodeWriter.cs

```diff
@@ -17,7 +17,7 @@ internal class BlazorDesignTimeNodeWriter : BlazorNodeWriter
     {
         private readonly ScopeStack _scopeStack = new ScopeStack();
 
-        private readonly static string DesignTimeVariable = "__o";
+        private static readonly string DesignTimeVariable = "__o";
```

Comment:
I think VS did this. They have a new feature that does 'cleanup' which I have turned on. So if you notice other stylistic changes like this, it's just VS doing it's thing.